### DISCUSSION TITLE
Feature Request: Resize Image Dimensions

### DISCUSSION THREAD

**Discussion by mithundsouza** [PaulWoitaschek/Voice#1423] [Ideas]
Hi
First of all, thank you for making this wonderful little app. Absolutely love it and have been using it almost everyday since I got it. Love the simplicity and all the features the app offers. It's just perfect for audiobook listening. 

One small feedback/request to passify the OCD in me. Is there any possibility to resize the image dimensions selected so they fit nicely on a portrait view of the phone. Right now, we get to select a 1:1 cover which crops the sides off in the full screen view in portrait mode. If there's a way to fix that, nothing like it. 
Some image refs
Portrait Fullscreen
![screenshot_20171127-085813](https://user-images.githubusercontent.com/16534463/33249565-6283884c-d351-11e7-87d8-c5b6e931fc81.jpg)

Landscape Full-screen
![screenshot_20171127-085834](https://user-images.githubusercontent.com/16534463/33249584-79fe0e7a-d351-11e7-8ae6-6bdc3f63218a.jpg)

Thanks

**Comment by xylo** [comment]
When you first open the a book, the cover image is shown really tiny.  See screenshot.  After app switching the cover is scaled to the available heigth, but not to the available width.  Would be nice if the app could scale the image so that it's neither cropped in x- or y-direction.  The aspect ratio should be kept of course.

![Screenshot_20220418-120915_Voice](https://user-images.githubusercontent.com/577869/163793975-787bf680-3d32-4afa-a6f4-7373d43db9d1.jpg)

**Comment by xylo** [comment]
Duplicates: #919, #992
